### PR TITLE
Ffmpeg 5 & prereqs

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=ffmpeg_5 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/ffmpeg.rb [hunk 1]
```diff
@@ -13,14 +13,14 @@ class Ffmpeg < Package
   binary_url({
     aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ffmpeg/4.4.1_armv7l/ffmpeg-4.4.1-chromeos-armv7l.tpxz',
      armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ffmpeg/4.4.1_armv7l/ffmpeg-4.4.1-chromeos-armv7l.tpxz',
```
Comment (uberhacker) [inline packages/ffmpeg.rb:15]: No arm build?

Comment (satmandu) [reply packages/ffmpeg.rb]: Fixed.

File: packages/swig.rb [hunk 2]
```diff
@@ -3,34 +3,35 @@
 class Swig < Package
   description 'Simplified Wrapper and Interface Generator'
   homepage 'http://www.swig.org'
-  version '4.0.1'
+  version '4.0.2'
   license 'GPL-3, BSD and BSD-2'
   compatibility 'all'
-  source_url 'http://prdownloads.sourceforge.net/swig/swig-4.0.1.tar.gz'
-  source_sha256 '7a00b4d0d53ad97a14316135e2d702091cd5f193bb58bcfcd8bc59d41e7887a9'
+  source_url 'http://prdownloads.sourceforge.net/swig/swig-4.0.2.tar.gz'
```
Comment (uberhacker) [inline packages/swig.rb:9]: Change to `https://downloads.sourceforge.net/project/swig/swig/swig-4.0.2/swig-4.0.2.tar.gz`?  IOW, use https.

Comment (satmandu) [reply packages/swig.rb]: Done.

**Pull request merged**